homeassistant/components/mochad/light.py

```diff
@@ -1,16 +1,17 @@
 """Support for X10 dimmer over Mochad."""
 import logging
 
+from pymochad import device
 import voluptuous as vol
 
+from homeassistant.components import mochad
```

Comment:
Intra package imports should be relative. Probably import the individual names from the module instead of the whole module/package.

Reply:
So from `. import DOMAIN, ...` and so on?

Reply:
Yes! Thanks!

Reply:
Sure thing! No problem!